templates/verify.txt

```diff
@@ -1,6 +1,6 @@
-{{t "Almost there"}}
+{{t "Welcome!"}}
```

Comment:
The exclamation adds a personal touch, and one is not excessive. A quick scan of verification emails from accounts this year include one is almost all of them somewhere in the email (including a new Gmail address). Let's keep it in.